README.md

```diff
@@ -20,3 +20,18 @@ auth:
     domainSuffix: 'sample.local'
     groupName: 'npmWriters' # optional
 ```
```

Comment:
Hey, Thanks for the pull request should we add documentation to the README.rd around the comment you made the the pull request and with the Scoped packages and restrictions to unpublish?

Reply:
I don't think its needed since it's functionality which is core to Verdaccio itself.